internal/cluster/cluster.go

```diff
@@ -1833,3 +1837,25 @@ func (m *Manager) UpdateFinalizingStage(ctx context.Context, clusterID strfmt.UU
 func (m *Manager) GetHostCountByRole(clusterID strfmt.UUID, role models.HostRole, suggested bool) (*int64, error) {
 	return common.GetHostCountByRole(m.db, clusterID, role, suggested)
 }
+
+func (m *Manager) HasBaseCapabilities(capabilities *installcfg.Capabilities, baselineCapability configv1.ClusterVersionCapabilitySet) bool {
+	return capabilities.BaselineCapabilitySet == baselineCapability
+}
+
+func (m *Manager) HasAdditionalCapabilities(capabilities *installcfg.Capabilities, includes, excludes []configv1.ClusterVersionCapability) bool {
```

Comment:
Wouldn't `HasAdditionalCapabilities(cap, includes) && !HasAdditionalCapabilities(cap, excludes)` work for you?

Reply:
Yes, much simpler, thank you!! Modified it!